spec/node-spec.ts

```diff
@@ -697,6 +697,27 @@ describe('node feature', () => {
       expect(code).to.equal(1);
     });
 
+    it('does allow --require in utility process of non-packaged apps', async () => {
+      const appPath = path.join(fixtures, 'apps', 'node-options-utility-process');
+      // App should exit with code 1.
+      const child = childProcess.spawn(process.execPath, [appPath]);
+      const [code] = await once(child, 'exit');
+      expect(code).to.equal(1);
+    });
+
+    it('does not allow --require in utility process of packaged apps', async () => {
```

Comment:
Oh lol cyclic dependency, this test will fail till https://github.com/electron/electron/pull/46210 is addressed. I will disable it in this PR and enable it as part of the other one.